noether_msgs/msg/PlaneSlicerRasterGeneratorConfig.msg

```diff
@@ -0,0 +1,6 @@
+float64 raster_spacing 	   # Offset between two rasters
```

Comment:
why don't all the configs have default values?

Reply:
That was mistake on my part, ros messages do not support default values.

Reply:
fixed.